jupyter_client/kernelspecapp.py

```diff
@@ -92,7 +91,7 @@ def start(self):
             raise
 
 class InstallNativeKernelSpec(BaseIPythonApplication):
-    description = """Install the native kernel spec directory for this Python."""
+    description = """[DEPRECATED] Install the IPython kernel spec directory for this Python."""
```

Comment:
But it's importing the new location, not the shim. And in any case, there should be a separate warning for the subcommand being deprecated as opposed to the import.